### ISSUE TITLE
Function keys don't work for flutter run on Windows

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#7804]
`flutter run --release` prints:
```
To repeat this help message, press "h" or F1. To quit, press "q", F10, or Ctrl-C.
```

However, pressing F1 or F10 does nothing on Windows.

**Comment by goderbauer** [comment]
This is blocked on https://github.com/dart-lang/sdk/issues/28984.

**Comment by goderbauer** [comment]
This has been fixed by rolling the Dart SDK to v1.23.0-dev9.0.
